site/management.md

```diff
@@ -285,20 +285,31 @@ rabbitmqctl set_user_tags full_access administrator
 
 ## <a id="oauth2-authentication" class="anchor" href="#oauth2-authentication">Authenticating with OAuth 2</a>
 
-RabbitMQ can be configured to use [JWT-encoded OAuth 2.0 access tokens](https://github.com/rabbitmq/rabbitmq-auth-backend-oauth2)
-to authenticate client applications and management UI users. When doing so, the management UI does
+RabbitMQ can be configured to use [JWT-encoded OAuth 2.0 access tokens](https://github.com/rabbitmq/rabbitmq-auth-backend-oauth2) to authenticate client applications and management UI users. When doing so, the management UI does
 not automatically redirect users to authenticate
 against the OAuth 2 server, this must be configured separately. Currently,
-only [UAA](https://github.com/cloudfoundry/uaa) is supported authorization server.
+RabbitMQ has been tested against the following Authorization servers:
```

Comment:
Replace "has been" above with "is"....use present tense where possible in documentation